src/extensions/mod.rs

```diff
@@ -593,10 +595,21 @@ impl fmt::Display for ReasonFlags {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct IssuingDistributionPoint<'a> {
+    pub distribution_point: Option<DistributionPointName<'a>>,
+    pub only_contains_user_certs: bool,
+    pub only_contains_ca_certs: bool,
+    pub only_some_reasons: Option<ReasonFlags>,
+    pub indirect_crl: bool,
+    pub only_contains_attribute_certs: bool,
+}
+
 pub(crate) mod parser {
     use crate::extensions::*;
     use crate::time::ASN1Time;
     use asn1_rs::{GeneralizedTime, ParseResult};
+    use der_parser::ber::BerObject;
```

Comment:
Thanks, that's interesting to know. I've left this as-is for now largely because I haven't had a chance to familiarize myself with the new style yet :-)